website/docs/usage/v3.md

```diff
@@ -390,10 +389,10 @@ Note that spaCy v3.0 now requires **Python 3.6+**.
 | `GoldCorpus`                                             | [`Corpus`](/api/corpus)                                                                                      |
 | `KnowledgeBase.load_bulk`, `KnowledgeBase.dump`          | [`KnowledgeBase.from_disk`](/api/kb#from_disk), [`KnowledgeBase.to_disk`](/api/kb#to_disk)                   |
 | `Matcher.pipe`, `PhraseMatcher.pipe`                     | not needed                                                                                                   |
-| `spacy init-model`                                       | [`spacy init model`](/api/cli#init-model)                                                                    |
+| `spacy init-model`                                       | [`spacy init vocab`](/api/cli#init.vocab)                                                                    |
```

Comment:
`init-vocab` instead of `init.vocab`

Reply:
Ah, then the anchor is wrong? The anchor should definitely be `#init-vocab`!

Reply:
The anchor is `#init-vocab`, right? But you're linking to `init.vocab` here?

Reply:
Arghhh I misread the diff 🤦‍♀️ Sorry!